azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/ArtifactHandlerImplV2.java

```diff
@@ -160,4 +182,44 @@ public void publishWarArtifact(final DeployTarget target, final File warArtifact
                 String.format("Failed to deploy war file after %d times of retry.", MAX_RETRY_TIMES));
         }
     }
+
+    private boolean isJavaSERuntime() throws MojoExecutionException {
+        return runtimeSetting != null &&
+            ((runtimeSetting.getOs().equals("windows") && this.project.getPackaging().equals("jar")) ||
```

Comment:
make a util method like `isWindows`, I see a lot of places using this piece of code.

Reply:
Change to enum to do the comparsion